### DISCUSSION TITLE
Annotating Video Files Located on a Local NAS

### DISCUSSION THREAD

**Discussion by hfreeman02** [biigle/core#839] [Q&A] [answered]
Hello all,

We are hoping to use Biigle to annotate video footage but are having issues with getting the data into the program. The files are stored locally on a NAS so we are thinking the best way to view them in Biigle is through storage requests. We loaded one in on a Friday (it took about 20min) and got the email Monday morning that it was approved! However, when we went to open it up it had a pop up saying "The video codec is invalid. Allowed codecs: h264, vp8, vp9, av1." Is there a way to work through this issue? Do we just have to alter the codecs of all the video files to match an allowed type?

Thanks!

**Comment by mzur** [comment, marked as answer]
Hi, your storage request seems to have been approved in error, sorry. Usually we check the video codec and in case it is not supported, we notify the user and reject the storage request.

Indeed, BIIGLE can only support videos that have a codec supported by major web browsers. These are H.264, VP8, VP9 or AV1. Your video is an MP4 with the H.265 codec.
